### DISCUSSION TITLE
Battery level for WSDCGQ11LM since release 1.27.0

### DISCUSSION THREAD

**Discussion by sebcaps** [Koenkk/zigbee2mqtt#13454] [Devices]
On my platform, installation of release  1.27.0 occured on August 02 at 12PM (using docker and watchover for automatic update).
Since this time all my [WSDCGQ11LM  ](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#xiaomi-wsdcgq11lm) battery devices moved from 100 % to about 45%. See below.

Just wondering if before this update battery values was 'wrong' and if this is a fix or improvement ? (ex [8499] ? )(https://github.com/Koenkk/zigbee2mqtt/issues/8499)
![image](https://user-images.githubusercontent.com/24374878/183396745-8692de6c-5712-4733-89d6-609c288fb6ce.png)

Thanks for your inputs.

**Comment by Koenkk** [comment]
Calculation has changed in 1.27.0, see https://github.com/Koenkk/zigbee2mqtt/issues/8499
